backend/apps/owasp/models/event.py

```diff
@@ -13,7 +13,14 @@ class Meta:
         db_table = "owasp_events"
         verbose_name_plural = "Events"
 
+    category = models.CharField(verbose_name="Category", max_length=100, default="", blank=True)
+    category_description = models.TextField(
+        verbose_name="Category Description", default="", blank=True
+    )
```

Comment:
This should be something like [project level ](https://github.com/OWASP/Nest/blob/66d71f976aff2fa7a0ba57cbdfe4ea48424fcd7d/backend/apps/owasp/models/project.py#L34) as they are shared for all events